src/Chapter4.hs

```diff
@@ -611,6 +630,13 @@ Implement the 'Monad' instance for our lists.
   maybe a few) to flatten lists of lists to a single list.
 -}
 
+sumList :: List (List a) -> List a
+sumList Empty = Empty
+sumList (Cons x xs) = append x (sumList xs)
```

Comment:
Great implementation!

Usually, `sumList` name would mean `+` somewhere. There is a standard function that does such inlining that is called `concat`. So you could name this one `concatList` or `InlineLists`. But this is just a side note 🙂